src/index.js

```diff
@@ -33,14 +36,31 @@ const client = createClient({
   ],
 })
 
-if (env('SENTRY_DSN') && env('ENABLE_SENTRY')) {
+const sentryEnabled = !!(env('SENTRY_DSN') && env('ENABLE_SENTRY'))
+
+if (sentryEnabled) {
   Sentry.init({
     dsn: env('SENTRY_DSN'),
     environment: getNetworkName(env('CHAIN_ID')),
     release: 'court-dashboard@' + env('BUILD'),
   })
 }
 
+let connectionAttempts = 0
+subscriptionClient.onConnected(() => (connectionAttempts = 0))
+
+// Check for connection errors and if reaches max attempts send error log to Sentry
+subscriptionClient.onError(err => {
+  const maxReconnectionAttempts = subscriptionClient.reconnectionAttempts
+
+  if (sentryEnabled && maxReconnectionAttempts === ++connectionAttempts) {
```

Comment:
Note that we don't really need the `sentryEnabled` flag:

<img width="756" alt="Screen Shot 2020-02-10 at 3 38 39 PM" src="https://user-images.githubusercontent.com/4166642/74159092-780a7280-4c1b-11ea-8ac0-4c93f16bbaf5.png">